test/test_cuda.py

```diff
@@ -580,9 +580,10 @@ def test_serialization_array_with_storage(self):
         self.assertTrue(isinstance(q_copy[0], torch.cuda.FloatTensor))
         self.assertTrue(isinstance(q_copy[1], torch.cuda.IntTensor))
         self.assertTrue(isinstance(q_copy[2], torch.cuda.FloatTensor))
-        self.assertTrue(isinstance(q_copy[3], torch.cuda.IntStorage))
+        self.assertTrue(isinstance(q_copy[3], torch.storage.TypedStorage))
+        self.assertTrue(isinstance(q_copy[3].storage, torch.cuda.ByteStorage))
```

Comment:
Is this the kind of test where y and y.storage() were stored in the same place, and it caused you the FC headaches (which you wrote a note about?) I wonder if there should be some sort of adjustment to the test itself, so it isn't trying to serialize both y and its storage at the same time, in one data structure.  OTOH this works just fine if you don't go backwards in time, so maybe not.

Reply:
Right, this is a non-FC case. But yeah it still works even though it isn't FC, so I think we should keep testing it